mlir/include/Driver/CompilerDriver.h

```diff
@@ -97,6 +99,8 @@ struct CompilerOutput {
     FunctionAttributes inferredAttributes;
     PipelineOutputs pipelineOutputs;
     size_t pipelineCounter = 0;
+    /// if the compiler reach the pass specified by startAfterPass.
+    bool reachTargetPass;
```

Comment:
I don't quite get this flag, how is it used?

Reply:
It is to check if the targeted pass is met. If yes, recompilation starts.

Reply:
I just renamed it into `isCheckpointFound`. Hopefully it is a bit more clear.

Reply:
Ah got it, the name was probably okay but now it clicked, thank you :)